products/looker.md

```diff
@@ -19,116 +19,87 @@ auto:
 
 # eol/esr dates on https://cloud.google.com/looker/docs/officially-supported-releases
 releases:
+-   releaseCycle: "24.12"
+    releaseDate: 2024-07-10
```

Comment:
Upcoming ESR release as per https://cloud.google.com/looker/docs/officially-supported-releases

```suggestion
    releaseDate: 2024-07-10
    lts: 2024-09-01
```

Reply:
Yes, I already set the LTS date, but at 2024-08-01